changelogs/fragments/6571-doc-random-string-crypto-secure.yml

```diff
@@ -0,0 +1,2 @@
+documentation:
```

Comment:
There is no changelog section called 'documentation'. See https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment for a list of all valid sections.

Documentation changes usually should not have a changelog fragment, so please delete this file.